src/PostgREST/Error.hs

```diff
@@ -253,6 +255,12 @@ instance JSON.ToJSON ApiRequestError where
   toJSON (ColumnNotFound relName colName) = toJsonPgrstError
     SchemaCacheErrorCode04 ("Could not find the '" <> colName <> "' column of '" <> relName <> "' in the schema cache") Nothing Nothing
 
+  toJSON (TableNotFound schemaName relName tbls) = toJsonPgrstError
+    SchemaCacheErrorCode05
+    ("Could not find relation '" <> schemaName <> "." <> relName <> "' in the schema cache")
```

Comment:
The "relation" term might be confused with "relationship". I think we can be more precise here and instead refer to "table" or "view", we have a `tableIsView :: Bool` field already.

Reply:
Although it doesn't make sense to discriminate table or view  if we don't know what we're looking for. Maybe you could omit the "relation" term altogether, or change it to "table or view".